src/api/zoops.ts

```diff
@@ -1,21 +1,38 @@
-import express, {Request, Response, NextFunction} from "express";
+import express, { Request, Response, NextFunction } from "express";
 import { PrismaClient } from "@prisma/client";
+import { prismaExclude } from "prisma-exclude";
 import requireUser from "../utils/requireUser";
 
 const prisma = new PrismaClient();
+const exclude = prismaExclude(prisma);
 
 const zoopsRouter = express.Router();
 
-
 // GET /api/zoops
 zoopsRouter.get("/", async (req, res, next): Promise<void> => {
   try {
-    const zoops = await prisma.zoop.findMany();
-    res.send({zoops});
+    const zoops = await prisma.zoop.findMany({
+      include: {
+        faves: {
+          include: {
+            faver: {
+              select: exclude("user", ["password"]),
```

Comment:
:o this is cool!

Maybe add an Issue to convert other uses of the excludePassword to use this instead?